libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -58,6 +61,9 @@ class MapboxTripSession(
     private val bannerInstructionEvent = BannerInstructionEvent()
     private val voiceInstructionEvent = VoiceInstructionEvent()
 
+    private var lastLocationUpdateTimeMillis: Long = 0
+    private val statusUpdateTimer = MapboxTimer().apply { restartAfterMillis = TimeUnit.SECONDS.toMillis(1) }
```

Comment:
why doesn't this use the STATUS_POLLING_INTERVAL?

Reply:
Because it is another constant and it can be different from STATUS_POLLING_INTERVAL. But in this particular case, values of this constants are equal, yes)

Reply:
STATUS_POLLING_INTERVAL just sounds like something that would determine the statusUpdateTimer 😄

Reply:
also doesn't look like STATUS_POLLING_INTERVAL is used